client/my-sites/stats/features/modules/stats-emails/tooltips.tsx

```diff
@@ -49,28 +49,32 @@ export const hasUniqueMetrics = ( uniqueValue: number, totalValue: number ) => {
 
 export const OpensTooltipContent: React.FC< { item: EmailStatsItem } > = ( { item } ) => {
 	const translate = useTranslate();
-	const opensUnique = parseInt( String( item.unique_opens ), 10 );
-	const opens = parseInt( String( item.opens ), 10 );
-	const opensRate = parseFloat( String( item.opens_rate ) );
-	const totalSends = parseInt( String( item.total_sends ), 10 );
-	const hasUniques = hasUniqueMetrics( opensUnique, opens );
+	const hasUniques = hasUniqueMetrics(
+		parseInt( String( item.unique_opens ), 10 ),
+		parseInt( String( item.opens ), 10 )
+	);
 
 	return (
 		<div className="stats-email__tooltip">
 			<div>
-				{ translate( 'Recipients: %(sends)d', {
-					args: { sends: totalSends },
+				{ translate( 'Recipients: %(sends)s', {
```

Comment:
Any danger this getting fuzzy-translated and us applying `%d` to where there now should be `%s`? That would break the UI I think.

Reply:
It's a good question! Thanks for bringing it up. We return null or undefined from `numberFormat` if the number is null/undefined. Otherwise it goes through `Intl.NumberFormat` (and in case of error, the original number back). So technically, the result is `null | undefined | string | number`, so yeah, this could break the UI if these properties aren't defined (according to the type though, they should be numbers).

I'll think this through. Perhaps add an extra String cast here or force a String return from `numberFormat`. WDYT?

Reply:
Oh I meant if in our translation pipeline the script notices that string is almost the same as before (just one char difference), marking them "fuzzy", and keeps returning the old string until translators translate the new version. I think we'd want the old string get not marked as "fuzzy", and fall back to English until translated.

Reply:
Ohhh right. It sounds possible. I think that's something we can check. I'll add the string freeze for now to see what that does to the strings.

@Automattic/i18n any thoughts on Mikael's note?

Reply:
The [similarity between both strings](https://mc.a8c.com/i18n/tools/gp-string-similarity.php?string_a=Recipients%3A+%25%28sends%29d&string_b=Recipients%3A+%25%28sends%29s) is 0.95, which would indeed use the fuzzy status, since the threshold is 0.8. Using `sendsCount` as a variable name brings it down to 0.74.

Reply:
Perfect. Thanks @dlind1 
Updated the strings and confirmed the similarity is less than 0.8. Needed a slightly longer word for the others.
We just wait for translations now :-)

Reply:
> We just wait for translations now :-)

Although I'm a bit confused. The status is no longer blocked after the updates. Also don't see a new "deliverable" from a8ci18n bot. The comment still links to the previous version: https://translate.wordpress.com/deliverables/17217319

Anything I need to do? removed & readded "string freeze" already